File: Python/단계별로 풀어보기/3. 반복문/#1110.py
N = int(input())

count = 0
value = N
while(True):
    ten = value // 10
    one = value % 10
    sum = ten + one

    new = one * 10 + sum % 10

    count+=1
    value = new

    if(new == N):
        break

print(count)

File: Java/단계별로 풀어보기/3. 반복문/#1110.java
import java.util.Scanner;

public class AdditionCycle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int x = sc.nextInt();

        int ten = x / 10;
        final int ten_answer = ten;
        int one = x % 10;
        final int one_answer = one;

        int count = 0;

        while(true) {
            int sum = ten + one;

            ten = one;
            one = sum % 10;

            count++;

            if(ten == ten_answer && one == one_answer) {
                System.out.println(count);
                break;
            }
        }
    }
}
